physics/Interstitials/UFS_SCM_NEPTUNE/GFS_rrtmgp_cloud_mp.F90

```diff
@@ -802,14 +798,18 @@ function cld_frac_XuRandall(p_lay, qs_lay, relhum, cld_mr, alpha)
        lambda = 0.50, & !
        P      = 0.25
 
-    clwt = 1.0e-6 * (p_lay*0.001)
+    clwt = 1.0e-8 * (p_lay*0.001)
     if (cld_mr > clwt) then
-       onemrh = max(1.e-10, 1.0 - relhum)
-       tem1   = alpha / min(max((onemrh*qs_lay)**lambda,0.0001),1.0)
-       tem2   = max(min(tem1*(cld_mr - clwt), 50.0 ), 0.0 )
-       tem3   = sqrt(sqrt(relhum)) ! This assumes "p" = 0.25. Identical, but cheaper than relhum**p
-       !
-       cld_frac_XuRandall = max( tem3*(1.0-exp(-tem2)), 0.0 )
+       if(relhum > 0.99) then
```

Comment:
@Qingfu-Liu You still need to have `cond_cfrac_onRH` in this logic, otherwise the Xu-Randall calculation always assumes this is True, but `cond_cfrac_onRH` is false for GFDLMP to RRTMGP/RRTMG coupling.

Reply:
@dustinswales Thanks. I will make the change

Reply:
@dustinswales  Just uploaded the changes. Thank you very much.